\hypertarget{classtestCaseHandler_1_1parser__SyntaxError}{}\section{test\+Case\+Handler\+::parser\+\_\+\+Syntax\+Error Class Reference}
\label{classtestCaseHandler_1_1parser__SyntaxError}\index{testCaseHandler::parser\_SyntaxError@{testCaseHandler::parser\_SyntaxError}}


Inheritance diagram for test\+Case\+Handler\+::parser\+\_\+\+Syntax\+Error\+:
% FIG 0


Collaboration diagram for test\+Case\+Handler\+::parser\+\_\+\+Syntax\+Error\+:
% FIG 1
\subsection*{Public Member Functions}
\begin{DoxyCompactItemize}
\item 
\mbox{\Hypertarget{classtestCaseHandler_1_1parser__SyntaxError_a5fcce8dde55a60d581f5ae4e88b492ab}\label{classtestCaseHandler_1_1parser__SyntaxError_a5fcce8dde55a60d581f5ae4e88b492ab}} 
const char $\ast$ {\bfseries what} ()
\end{DoxyCompactItemize}


\subsection{Detailed Description}


Definition at line 36 of file test\+Case\+Exception.\+h.



The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
my\+Libraries/test\+Case\+Analyser/src/test\+Case\+Exception.\+h\end{DoxyCompactItemize}
